docs/sources/panels/add-a-panel.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Add a panel"
+type = "docs"
+[menu.docs]
+identifier = "add-a-panel"
+parent = "panels"
+weight = 100
+draft = "true"
++++
+
+# Add a panel
+
+Panels allow you to show your data in visual form. This topic walks you through the most basic steps to build a panel.
+
+## 1. Add a panel to a dashboard
+
+1. Navigate to the dashboard you want to add a panel to.
+1. Click the **Add panel** icon.
+   
+     ![](/img/docs/panels/add-panel-icon-7-0.png)
+
+1. Click **Add new panel**.
+
+Grafana creates an empty graph panel with your default data source selected.
+
+## 2. Edit panel settings
+
+While not required, we recommend that you add a helpful title and description to your panel. You can use variables you have defined in either field, but not global variables.
+
+![](/img/docs/panels/panel-settings-7-0.png)
+
+**Panel title -** Text entered in this field is displayed at the top of your panel in the Panel editor and in the dashboard.
+
+**Description -** Write a description of the panel and the data you are displaying. Pretend you are explaining it to a new user six months from now, when it is no longer fresh in your mind. Future editors (possibly yourself) will thank you.
```

Comment:
This is optional and shown in a tooltip in upper left corner (an info icon is shown after adding a description). You can use markdown in the description with links.